tests/resource_test.py

```diff
@@ -17,3 +18,22 @@ def test_static_files():
         "myapp-schema not in list={}",
         x,
     )
+
+
+def test_render_resource():
+    from sirepo import resource
+    from pykern import pkunit
+
+    d = pkunit.data_dir()
+    pkunit.file_eq(
+        d.join("expect.py"),
+        actual_path=resource.render_resource(
+            "actual.py",
+            d,
+            pkunit.work_dir(),
+            PKDict(
+                x="x",
```

Comment:
Use something that's not the same thing as the key, e.g. x=1. Also, there really only needs to be one value to make sure the parameter gets passed. The fact that it has two values complicates the test without adding value.